### ISSUE TITLE
Error naming inconsistent

### ISSUE THREAD

**Issue by Chemin1** [ElektraInitiative/libelektra#2901]
The names of the errors in the files doc/decisions/error_codes.md and doc/dev/error-categorization.md
do not fit together _exactly_, but it is not a real problem => low priority label.

C01310 is called Assertion in error_codes and Internal in error-categorization
C01330 is called Plugin is Broken in error_codes and Plugin Misbehaviour in error-categorization

**Comment by markus2330** [comment]
Thank you for finding out and reporting this problem!

"low priority" is only for something which is okay if it is not fixed at all. But this is definitely not the case here.

**Comment by Piankero** [comment]
I fixed it on my branch which I will make a PR once it is ready for it (https://github.com/Piankero/libelektra/commit/f8fb57a27d123b7fc1f62f0830150e37014c06f0)

**Comment by markus2330** [comment]
It is much preferred to have several smaller PRs than a monster PR which takes forever to review and merge.
